.gitignore

```diff
@@ -1,2 +1,3 @@
 /lib/
 /node_modules/
+pnpm-lock.yaml

```

Comment:
I don't like adding environment specific files to ignore files, otherwise ignore files become polluted with every system/IDE specific file under the sun. You should ignore these globally in your environment instead: https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer